lib/ansible/modules/cloud/kubevirt/kubevirt_pvc.py

```diff
@@ -340,6 +369,29 @@ def _parse_cdi_source(self, _cdi_src, metadata):
             if 'certConfigMap' in src_spec:
                 annotations['cdi.kubevirt.io/storage.import.certConfigMap'] = src_spec['certConfigMap']
 
+    def _wait_for_creation(self, resource, uid):
+        return_obj = None
+        use_cdi = True if self.params.get('cdi_source') else False
+
+        for event in resource.watch(namespace=self.namespace, timeout=self.params.get('wait_timeout')):
+            entity = event['object']
+            metadata = entity.metadata
+            if not hasattr(metadata, 'uid') or metadata.uid != uid:
+                continue
+            if entity.status.phase == 'Bound':
+                if use_cdi and hasattr(metadata, 'annotations'):
+                    import_status = metadata.annotations.get('cdi.kubevirt.io/storage.pod.phase')
+                    if import_status == 'Succeeded':
+                        return_obj = entity
+                        break
+                else:
+                    return_obj = entity
+                    break
+            elif entity.status.phase == 'Failed':
+                self.fail_json(msg="Failed to create PersistentVolumeClaim")
```

Comment:
I would raise an exception here, we use fail_json at the end

Reply:
done